modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -52,6 +52,8 @@ export interface BuildTokenEnablementOptions extends PrebuildTransactionOptions
   enableTokens: TokenEnablement[];
 }
 
+export type ApiVersion = 'lite' | 'full' | undefined;
```

Comment:
i think the extra `| undefined` is redundant.

you can declare a property as being optional using something like:
```
apiVersion?: ApiVersion
```